shell/platform/windows/win32_dpi_helper.h

```diff
@@ -28,17 +28,35 @@ class Win32DpiHelper {
   // Wrapper for OS functionality to return the DPI for |HWND|
   UINT GetDpiForWindow(HWND);
 
+  // Wrapper for OS functionality to return the DPI for the System. Only used if
+  // Per Monitor V2 is not supported by the current Windows version.
+  UINT GetDpiForSystem();
```

Comment:
Similar to the comment below: is there a reason to expose this separately rather than making it an internal detail of GetDpiForWindow?

Reply:
GetDpiForWindow is also the name of the win32 api function, which is specific for Per Monitor V2. I didn't want to mix the names and create confusion on what it does. I could just change it to GetDpi, and under the hood, do either ..DpiForWindow/System accordingly. What do you think?

Reply:
SGTM.

Reply:
Done